build.gradle

```diff
@@ -324,6 +331,11 @@ task generateJournalAbbreviationList(type: JournalAbbreviationConverter) {
     outputDir = file("src/main/resources/journals")
 }
 
+task fixCheckstyle {
+   dependsOn(rewriteRun)
+}
+
```

Comment:
Please only a single empty line for separation :p

Reply:
You and your obsession with empty lines! There is no checkstlye rule for the build.gradle :-P